src/SFA.DAS.ApprenticePortal.SharedUi/Menu/NavigationSection.cs

```diff
@@ -11,7 +11,8 @@ public enum NavigationSection
         Login,
         PersonalDetails,
         TermsOfUse,
-        Registration
+        Registration,
+        ApprenticeFeedback
```

Comment:
I like to add trailing commas to avoid future diffs higlighting this line.

I appreciate this wasn't done already in this file!